circus-train-api/src/main/java/com/hotels/bdp/circustrain/api/conf/ReplicationMode.java

```diff
@@ -19,6 +19,8 @@
 
   FULL,
   METADATA_MIRROR,
-  METADATA_UPDATE;
+  METADATA_UPDATE,
+  FULL_OVERWRITE;
```

Comment:
Move this up to below `FULL` - they feel more "related" and I don't think we use the ordering of this enum for anything.